portal.go

```diff
@@ -1946,7 +1956,7 @@ func (portal *Portal) convertiMessage(msg *imessage.Message, intent *appservice.
 	return attachments
 }
 
-func (portal *Portal) handleNormaliMessage(msg *imessage.Message, dbMessage *database.Message, intent *appservice.IntentAPI) {
+func (portal *Portal) handleNormaliMessage(msg *imessage.Message, dbMessage *database.Message, intent *appservice.IntentAPI, mxID *id.EventID) {
```

Comment:
mxid is treated as a single word so should have consistent capitalization (`mxid` in this case, `MXID` is also valid in other cases)

Reply:
Happy to correct tomorrow. I was reading it as two separate words, but am unfamiliar with most matrix terminology.